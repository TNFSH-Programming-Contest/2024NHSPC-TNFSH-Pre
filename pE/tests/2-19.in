19
989984 986156 957643 987707 905191 971959 929164 983999 978808 916617 996188 930414 944824 958021 954445 957576 986512 995902 906277 1
