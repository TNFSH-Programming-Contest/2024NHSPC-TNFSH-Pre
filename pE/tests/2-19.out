15252098849095